source/docs/appendices/migration-guides.md

```diff
@@ -15,6 +15,25 @@ Node.js 10.x [reaches its end of life in the end of April 2021](https://nodejs.o
 
 We also recommend to use Node.js 14.x as this version is also supported and it is the current active Long Term Support Node.js release.
 
+### Wishlist Provider
+
+In this release we have implemented a wishlist provider that takes care of:
+
+- Checking if the wishlist feature is enabled
+- Loading wishlist
+- Checking if product is in wishlist
+
+By doing so we have unified some queries related to the wishlist moved them to the provider. It is highly recommended you switch to using the wishlist provider for the functionalities it provided instead of relying on querying the data directly. This will also help reduce the number of requests you do to the server (like in the case of checking if product is in wishlist). We have also updated the `withIsProductInWishlist` HOC to use the provider so continuing to use it is ok.
```

Comment:
```suggestion
By doing so, we have unified some queries related to the wishlist moved them to the provider. If you use this feature or have overridden related components, we highly recommend you to update them. Leveraging the wishlist provider instead of querying the data directly reduces the number of client / server requests and make your application more performant.
```